Mention indices counting from 0. (#2072)
enkidevs/curriculum: java/fundamentals/operators-ii/from-is-included-while-to-is-excluded.md

As a convention, when specifying a range of values, the first value 'from' is included while the last value 'to' is excluded. 

For example in `String.substring(from, to)`, the character at position 'from' will be included in the substring, while the character 'to' will not.

> Indices count from 0, as common in many programming languages.  

Reviewer: Thanks for the suggestion @Robaert-codes. How does this look? (the `>` formats the text in-app as being a quote)

 ```suggestion
> **Remember:** As with most programming languages, indexes in Java start from 0.
```
